xtask/src/build_docker_image.rs

```diff
@@ -249,15 +243,13 @@ pub fn build_docker_image(
         if engine.kind.supports_pull_flag() {
             docker_build.arg("--pull");
         }
+        let untagged_name = format!("{repository}/{}", target.name);
```

Comment:
what does `untagged` mean?

Reply:
I think this could be renamed to just `image_name`

Reply:
It's for podman: the name without the tag. `ghcr.io/cross-rs/x86_64-unknown-linux-gnu` is the untagged name, and `ghcr.io/cross-rs/x86_64-unknown-linux-gnu:main` is the tagged name. Podman doesn't accept tags for the `--cache-from` (sigh).

Reply:
> I think this could be renamed to just `image_name`

I intentionally avoided that because `target.image_name` adds the tag to the image, which is then confusing.

Reply:
maybe `base_name`?